compositor_render/src/transformations/layout/apply_layouts.wgsl

```diff
@@ -12,9 +12,10 @@ struct VertexOutput {
 
 
 struct Layout {
-    position_transformation: mat4x4<f32>,
-    texture_id: i32,
+    vertices_transformation: mat4x4<f32>,
+    texture_coord_transformation: mat4x4<f32>,
```

Comment:
Technically we need only `mat3x3<f32>`, but the memory layout of `mat3x3` requires adding padding to 16 bytes after each column, so I used `mat4x4` for convenience.